### PR TITLE
PEP601: Add rejection notice

### PR DESCRIPTION
**Pull request by dpgeorge**

### CODE REVIEW

File: pep-0601.txt [hunk 1]
```diff
@@ -3,12 +3,24 @@ Title: Forbid return/break/continue breaking out of finally
...
 Created: 26-Aug-2019
 Python-Version: 3.8
-Post-History: 26-Aug-2019
+Post-History: 26-Aug-2019 23-Sep-2019
+Resolution: https://discuss.python.org/t/pep-601-forbid-return-break-continue-breaking-out-of-finally/2239/32
+
+Rejection Note
+==============
+
+This PEP was rejected by the Steering Council by a vote 4/4.
```
Comment (isidentical) [inline pep-0601.txt]: IMHO `by a vote of 4/4` is more clear.

Comment (dpgeorge) [reply pep-0601.txt]: Ok, fixed.

**Comment by dpgeorge** [comment]
Let me know if this needs to be squashed before it can be merged.

**Comment by Rosuav** [comment]
No, it's fine, we squash as we merge. Looks good to me; anyone else have comments before merging?

**Comment by gvanrossum** [comment]
Thanks for being a good sport, and good luck with MicroPython. I won't hold it against you if you don't implement these. :-)

**Comment by dpgeorge** [comment]
This was my first PEP, a good experience!

MicroPython supports return and break in a finally.  I'll try to fix continue, otherwise make it a SyntaxError.

**Pull request merged**